test/ui/viewSpecUITest.js

```diff
@@ -0,0 +1,42 @@
oaches, with room to learn, develop and grow. You will manage, coach and develop a small number of staff, with a focus on managing employee performance and assisting in their career development. You’ll also provide direction and leadership for your team as you solve challenging problems together. MINIMUM (ESSENTIAL) REQUIREMENTS: • Proven experience being accountable for different sizes and shapes of technology delivery challenges, e.g. services project, multi-team programme, packaged product. • Able to simply and clearly communicate technical design in conversation, documentation and presentations. • Able to prioritise their time across multiple major projects particularly when working to deadlines. • Has successfully led & delivered software designs for multi-tiered modern software applications. • Understands whole solution architecture concepts and can communicate and negotiate these with senior stakeholders. • Can prioritise non-functional concerns for customers and has experience incorporating these into the application design. • Has an engineering background, allowing effective communication with, assurance of and leadership of development teams. • Is focused on improvement of process, people and use of technology. • Has experience with public cloud platforms, such as AWS and Azure, including SaaS and PaaS offerings. • Has pro-actively developed business across an account with sales and account managers. • Can build credibility and communicate effectively with C-level stakeholders. • Understands commercial implications of design decisions and has influenced the commercial success of a product. • We are passionate about developing people – a demonstrated ability in managing, coaching, and developing junior members of your team and wider community. DESIRABLE: • Actively shares their thoughts and views on technology. • Has participated in technology communities. WHO YOU ARE Our vision is to enable outstanding people to create digital solutions that have a positive impact on people’s lives. Our values are not abstract; they are the behaviours we expect from each other every day and underpin everything that we do. We expect everyone to display our values by being determined in how obstacles are overcome; honest when dealing with others; respectful of how you treat others; creative to find solutions to complex problems and cooperative by sharing information, knowledge and experience. These values, applied collectively, help to produce an outstanding Kainos person, team and culture. ABOUT US At Kainos we use technology to solve real problems for our customers, overcome big challenges for businesses, and make people’s lives easier. We build strong relationships with our customers and go beyond to change the way they work today and the impact they have tomorrow. Our two specialist practices, Digital Services and Workday, work globally for clients across healthcare, commercial and the public sector to make the world a little bit better, day by day.')
```

Comment:
What do you think of changing this assertion a bit. You think we could try checking that model contains a sentence from this paragraph? 

```suggestion
        chai.assert.isTrue(value.contains('As a Principal Architect (Principal) in Kainos'))
```